config/exporters/monitoring/prometheus/base/prometheus-exporter-service.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: exporter-cluster-role
+rules:
+- apiGroups: ['grafana.integreatly.org']
+  resources: ['grafanas']
+  verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: exporter-cluster-role-binding
+subjects:
+- kind: ServiceAccount
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: exporter-cluster-role
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: exporter-service
+  namespace: appstudio-exporter-service-prometheus
+spec:
+  ports:
+  - name: https
+    port: 8090
+    targetPort: https
+  selector:
+    app: prometheus-exporter
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: exporter-service-deployment
+  namespace: appstudio-exporter-service-prometheus
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prometheus-exporter
+  template:
+    metadata:
+      labels:
+        app: prometheus-exporter
+    spec:
+      serviceAccountName: exporter-sa
+      containers:
+      - name: prometheus-exporter
+        image: quay.io/redhat-appstudio / o11y-prometheus-exporters:latest
```

Comment:
```suggestion
        image: quay.io/redhat-appstudio/o11y-prometheus-exporters:latest
```

Reply:
Already pushed the change, but due to Github issues it took sometime to reflect the change.